### PR TITLE
Drafting a general pitch to send to potential hosts.

### PR DESCRIPTION
**Pull request by gvwilson**
1.  Incorporate @karinlag's draft as `workshops/pitch.html` (with edits).
2.  Reference the pitch from operations pages.
3.  Move the graphs of recent stats into `img/graphs`.
4.  Edit pages that refer to them to include them consistently.
5.  Delete older stats images (no longer needed).

### CODE REVIEW

**Comment by gvwilson** [comment]
@karinlag @jduckles please review

File: workshops/pitch.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  In many cases they end up spending more time than necessary to create results
+  whose reliability and reproducibility is unknown.
+</p>
+<p>
+  The aim of Software Carpentry is to teach researchers the fundamental skills that will help them be more productive
+  while producing work of higher quality.
+  We do that by teaching two day workshops,
+  where students learn how to automate tasks using the Unix shell,
+  how to track and share work using version control,
+  and how to write software in Python, R, or MATLAB that is readable, reusable, and reliable.
```
Comment (karinlag) [inline workshops/pitch.html]: or Matlab, which is readable, ...

**Comment by karinlag** [comment]
Looks great to me (but then again, it would, wouldn´t it :))

**Comment by jduckles** [comment]
Looks good. I'm going to probably replace the graphics with some R scripts I've been toying with, but this works for now.

**Comment by katyhuff** [comment]
Recommend including a citation in the Results section.

**Pull request merged**